### DISCUSSION TITLE
How to add `lang` attribute on the `html` tag?

### DISCUSSION THREAD

**Discussion by vedxp** [nuxt/nuxt#18784] [Questions] [answered]
Is it possible to add `lang` attribute on the `html` tag somehow?
I'm not aware of any way to do so.

I got to know when I used [html validator plugin](https://html-validator.nuxtjs.org/) and the plugin gave me this validation error:
> ![Screenshot_2023-02-05-20-27-18-53_84d3000e3f4017145260f7618db1d683~2](https://user-images.githubusercontent.com/87353286/216828049-d34b1269-764f-47f5-bb83-7fae465da514.jpg)

**Comment by AloisSeckar** [comment, marked as answer]
`useHead` with `htmlAttrs` option - https://nuxt.com/docs/api/composables/use-head

```js
useHead({
    htmlAttrs: {
        lang: 'en'
    }
})
```

**Reply by vedxp** [reply]
> `useHead` with `htmlAttrs` option - https://nuxt.com/docs/api/composables/use-head
> 
> ```js
> useHead({
>     htmlAttrs: {
>         lang: 'en'
>     }
> })
> ```

Oh, thank you so much.

Could you also please confirm me if I can use `head` inside `nuxt.config.js` like this to do the same?
```js
app: {
    head: {
      htmlAttrs: {
        lang: "en",
      },
    },
  },
```

**Reply by AloisSeckar** [reply]
> 
> Could you also please confirm me if I can use `head` inside `nuxt.config.js` like this to do the same?
> 

yes, it should work as well...this is the minimal `nuxt.config.js` that does the thing for me in Nuxt 3:

```js
export default defineNuxtConfig({
  app: {
    head: {
      htmlAttrs: {
        lang: 'en',
      },
    },
  },
})
```

**Reply by vedxp** [reply]
> > Could you also please confirm me if I can use `head` inside `nuxt.config.js` like this to do the same?
> 
> yes, it should work as well...this is the minimal `nuxt.config.js` that does the thing for me in Nuxt 3:
> 
> ```js
> export default defineNuxtConfig({
>   app: {
>     head: {
>       htmlAttrs: {
>         lang: 'en',
>       },
>     },
>   },
> })
> ```

Woah, thank you so much for confirming.

And I appreciate your efforts in helping me :)

**Comment by vedxp** [comment]
@AloisSeckar Hello sir, additionally I'm having difficulty regarding [this](https://github.com/nuxt/content/discussions/1886) and would appreciate your help in it :)
